docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,219 @@
+2. If you are pushing a last typo fix on a PR, it is okay to add `[skip ci]` in the commit message. This will make the commit skip the Travis build. Do not do this if 
+it is the first commit or if there are any changes to the actual code. 
+
+## 2. Issues
+Issues are used to bring the team's attention to any bugs or feature requests that you come across. When you make an issue, there are 4 options: bug fix, feature request, security vulnerability, or a blank format. You will probably either be making a bug fix or feature request. In these cases, there will be a template to fill out.
+
+Every PR should have a related issue that it is fixing. If you are going to be making a 
+change, make an issue for it and assign yourself to be the one to fix it.
+
+Be sure to observe and follow conventions for how Issues should be named and described and how to ping other people for input.
+ 
+Our team tends to self-assign issues. If you would like to take on a non-trivial issue, please ask us for input early and often.
+
+Don’t just follow the template blindly. For example, remove the “Brief summary of issue” header and put your summary of the issue there.
+Don’t leave obvious template words like “(if applicable)” in your issue.
+        
+## 3. Pull Requests
+Once you have created a new branch and pushed an initial commit to it, you will see the option to make a pull request on GitHub. 
+Make sure that the branch you are merging into is the branch that you checked-out when you first started making changes, such as main or a release-specific branch.
+
+PRs must be reviewed according to this [checklist](https://github.com/aces/Loris/wiki/Code-Review-Checklist), and must be approved by two reviewers before they are merged. Don’t forget to think about documentation updates like the TestPlan, changelog, and developer guide updates -- these must accompany your PR.
+
+Make sure to note special rules for SQL patches. Note that changes to Raisinbread data do not need patches, because Rida manages it.
+
+**Do not** review or approve your own PR. Your PR must be reviewed and approved by two other developers before it can be merged. 
+
+**Important:** Do not merge your own PR or any other PR. After a PR has been reviewed and approved, the code is merged by Dave, the repository maintainer. It will be merged eventually. If your PR is on a release branch and has been ready to merge for a few days, you can add a comment that it is ready to merge. 
+
+It is your responsibility to be on top of your PRs and any Issues that need follow-up.  This includes answering questions and comments, acknowledging and making change requests, and requesting re-review after you’ve addressed someone’s change requests. 
+
+This is important for the Team’s productivity -- it prevents PRs from lagging and prevents confusion as to why a PR has stalled or if you’ve seen someone’s comments.
```

Comment:
This sentence should be part of the previous paragraph, not a separate paragraph.. otherwise it's not immediately obvious what "this" is referring to. (Also "the Team" isn't a proper noun)